model/db.js

```diff
@@ -37,9 +37,9 @@ async function initDb() {
   CREATE TABLE IF NOT EXISTS advertisedCarRide (
     driver INTEGER REFERENCES appUserAccount(userID),
     car VARCHAR(10) REFERENCES userOwnsAcar(licensePlate),
-    date DATE NOT NULL,
+    date DATE NOT NULL CHECK ((date > current_date) OR (date = current_date AND time < current_time)),
```

Comment:
`time < current_time` should be `time > current_time`

Reply:
Nice catch